src/dsp_tools/xmllib/helpers.py

```diff
@@ -28,6 +28,47 @@ def create_label_to_name_list_node_mapping(
 
     Returns:
         a dictionary of the form {label: name}
+
+    Examples:
+        ```json
+        "lists": [
+            {
+                "name": "listName",
+                "labels": {
+                    "en": "List",
+                    "de": "Liste"
+                },
+                "comments": {
+                    "en": "This is a list"
+                },
+                "nodes": [
+                    {
+                        "name": "n1",
+                        "labels": {
+                            "en": "Node 1",
+                            "de": "Knoten 1"
+                        }
+                    },
+                    {
+                        "name": "n2",
+                        "labels": {
+                            "en": "Node 2",
+                            "de": "Knoten 2"
+                        }
+                    }
+                ]
+            }
+        ]
+        ```
+
+        ```python
+        result = xmllib.create_label_to_name_list_node_mapping(
+            project_json_path="project.json",
+            list_name="listName",
+            language_of_label="de",
+        )
+        # -> { "Knoten 1": "n1", "Knoten 2": "n2" }
```

Comment:
I asked Noemi about the display of the result and she was happy with this.
I also considered the more verbose but also more explicit:
`# result = {}`
This would be fine by me too if you prefer it.